### PR TITLE
framework/task_manager: Refactoring for task manager core

### PR DESCRIPTION
**Pull request by jeongchanKim**
1. taskmgr_dealloc_cb_info will release information about stop and exit callback
2. taskmgr_handle_tcb will open taskmgr_drv and call ioctl for task manager
3. add close for g_taskmgr_fd
4. unify the macro of stop, exit and user data callback information

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [2f16828c01700b7aeb9c7c91a540aaab84b11331] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [2f16828c01700b7aeb9c7c91a540aaab84b11331] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [2cc3c792e38989ee9c85639f2b52e7144e1b20aa] -

**Comment by seinfra** [comment]
Target : [2cc3c792e38989ee9c85639f2b52e7144e1b20aa] -

**Comment by seinfra** [comment]
Target : [e1d731f54e9dd7a450212cade4179c9fdfe35fd3] -

**Comment by seinfra** [comment]
Target : [e1d731f54e9dd7a450212cade4179c9fdfe35fd3] -

**Comment by seinfra** [comment]
Target : [3273f667c57e30940b9483098a2f973a15283d53] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [3273f667c57e30940b9483098a2f973a15283d53] - Code Rule Check OK.

File: framework/src/task_manager/task_manager_core.c [hunk 1]
```diff
@@ -71,26 +71,10 @@ static int task_manager_pid;
 #define TYPE_CANCEL      3
 #define TYPE_EXIT        4
 
-#define CB_FUNC_OF(X)             (X)->cb
-#define CB_DATA_OF(X)             (X)->cb_data
+#define CB_FUNC_OF(X)             ((tm_termination_info_t *)X)->cb
```
Comment (jeongarmy) [inline framework/src/task_manager/task_manager_core.c]: How about changing X to something like 'handle' for readability?

**Pull request merged**